src/test/java/bit/javaoop/EmployeeNumberManagerTest.java

```diff
@@ -7,23 +7,30 @@
 class EmployeeNumberManagerTest {
 
     @Test
-    void isSatisfied() {
-        EmployeeNumberManager satisfiedManager = new EmployeeNumberManager("Marek", 25_000, 0);
-        EmployeeNumberManager emptySpotManager = new EmployeeNumberManager("Darek", 25_000, 1);
-        EmployeeNumberManager lowSalaryManager = new EmployeeNumberManager("Jarek", 20_000, 0);
-
+    void shouldBeSatisfied() {
+        Manager satisfiedManager = new EmployeeNumberManager("Marek", 25_000, 0);
         assertTrue(satisfiedManager.isSatisfied());
+    }
+
+    @Test
+    void shouldBeNotSatisfied() {
+        Manager emptySpotManager = new EmployeeNumberManager("Darek", 25_000, 1);
+        Manager lowSalaryManager = new EmployeeNumberManager("Jarek", 20_000, 0);
         assertFalse(emptySpotManager.isSatisfied());
         assertFalse(lowSalaryManager.isSatisfied());
     }
 
     @Test
-    void canHire() {
-        EmployeeNumberManager noSpotsManager = new EmployeeNumberManager("Marek", 25_000, 0);
-        EmployeeNumberManager freeSpotManager = new EmployeeNumberManager("Darek", 25_000, 1);
+    void shouldHire() {
+        Manager freeSpotManager = new EmployeeNumberManager("Darek", 25_000, 1);
         Employee employee = new Employee("Jarek", 10_000);
+        assertTrue(freeSpotManager.canHire(employee));
+    }
 
+    @Test
+    void shouldNotHire() {
```

Comment:
Miło się patrzy na te nazwy :)
Jeszcze mógłbyś rozważyć dodanie do nazwy testu słówka `when` - przykład: `shouldNotHireWhenOutOfSpots` albo `shouldNotHireWhenNoFreeSpots`. (to drugie chyba lepsze, nie mam dziś głowy do nazw :D). Wiesz o co chodzi, dodaje to do nazwy testu takie już po prostu ostateczne rozwianie wątpliwości co robi test. Mianowicie czytamy to tak: 
+ mamy nazwę klasy EmployeeNumberManagerTest - wiemy, że testy dotyczą tychże Managerów (swoją droga wieeeeelki plus za podział testów managerów na dwie klasy)
+ mamy `shouldNotHire` - wiemy, że Manager w tym teście nie powinen przyjąc nowego pracownika
+ mamy `whenNoFreeSpots` - wiemy z miejsca jaki jest condition działania testu - brak wolnych spotów/slotów

Ogólnie właśnie o to chodziło z tymi testami. Takie nazewnictwo daje nam natychmiastowe ogarnięcie sytuacji. Nie trzeba kompletnie zagłębiać się w ciało metody, żeby wiedzieć co test ma robić. Jeśli się wywali, to od razu wiemy co zawiodło. Dlatego tak często stosuje się tę konwencję w pracy :)

PS. zapomniałeś o komentarzach!! :D `//given` `//when` `//then` :)
```
//given
Manager noSpotsManager = new EmployeeNumberManager("Marek", 25_000, 0);
Employee employee = new Employee("Jarek", 10_000);
//when
boolean result = noSpotsManager.canHire(employee);
//then
assertFalse(result);
```
Trochę wymusiłem ten blok `//when` z tym booleanem - wcale nie musi tak być w razie czego. Blok `//when` może być pusty - dopuszczam taką możliwość. Acz lepiej jak zawsze wiadomo CO się testuje - to nam mówi blok `//when`.